lib/jsonld_serializers.js

```diff
@@ -121,7 +140,7 @@ class JsonLdItemSerializer extends JsonLdSerializer {
         if (identifierTypes[prefix]) {
           var apiProp = identifierTypes[prefix]
           if (!stmts[apiProp]) stmts[apiProp] = []
-          stmts[apiProp].push(value)
+          if (stmts[apiProp].indexOf(value) < 0) stmts[apiProp].push(value)
```

Comment:
Ensure values are unique by checking first that it doesn't already exist before adding.